### DISCUSSION TITLE
Hub pricing

### DISCUSSION THREAD

**Discussion by RaananHadar** [microsoft/PlanetaryComputer#87] [Q&A]
It is unclear from the docs whether the Hub compute resources cost money. Can you please clear this up? thank you.

**Comment by TomAugspurger** [comment]
There's no cost for the Hub deployed at https://planetarycomputer.microsoft.com/compute.

If you [Deploy your own Hub](https://planetarycomputer.microsoft.com/docs/concepts/hub-deployment/), that would be billed to your Azure subscription.
